app/jobs/add_player_job.rb

```diff
@@ -3,6 +3,16 @@ class AddPlayerJob < ApplicationJob
 
   def perform(username)
     player = Syncer.add_player!(username)
-    SlackNotifier.send_message("Added #{username} as #{player.tag}")
+    SlackNotifier.send_message("Added #{username} as #{player.tag}. Syncing data...")
+
+    Syncer.sync!(notify: true, username: player.username)
```

Comment:
I was kinda a little bit worried about two syncs running at the same time (this + background job) but ... I guess it doesn't matter.